common/lib/xmodule/xmodule/modulestore/tests/sample_courses.py

```diff
@@ -2,12 +2,15 @@
 """
 The data type and use of it for declaratively creating test courses.
 """
-# used to create course subtrees in ModuleStoreTestCase.create_test_course
+BlockInfo = namedtuple('BlockInfo', 'block_id, category, fields, sub_tree')
 # adds to self properties w/ the given block_id which hold the UsageKey for easy retrieval.
 # fields is a dictionary of keys and values. sub_tree is a collection of BlockInfo
```

Comment:
This multi-line comment seems to have been mangled by isort, should probably move with the `BlockInfo` declaration.

Reply:
OK sure, I'll fix that real quick.